clients/python/lorax/client.py

```diff
@@ -155,7 +155,9 @@ def generate(
         payload = resp.json()
         if resp.status_code != 200:
             raise parse_error(resp.status_code, payload)
-        return Response(**payload[0])
+        if isinstance(payload, list):
+            return Response(**payload[0])
```

Comment:
This smells like bad design IMO. The REST API should be consistent and predictable. We should not be writing client-side hacks to account for inconsistency in the REST API.